app/liquid/shares.rb

```diff
@@ -0,0 +1,8 @@
+module Shares
+  def self.get_all(page)
+    Share::Button.where(campaign_page_id: page.id).inject({}) do |shares, button|
+      shares[button.sp_type] = button.sp_button_html
+      shares
+    end
+  end
+end
```

Comment:
is there a reason for this to be in the liquid directory? it seems like more of a fit for services or lib or models.